common/script/src/content/mystery-sets.js

```diff
@@ -0,0 +1,128 @@
+import {each, where} from 'lodash';
+import {flat as flattenedGear} from './gear/index';
+
+let mysterySets = {
+  201402: {
+    start: '2014-02-22',
+    end: '2014-02-28',
+    text: 'Winged Messenger Set'
```

Comment:
I don't know why we don't translate these at present. Make them into `translator` calls?